generators/entity-client4/templates/ionic/src/app/pages/entities/_entity.spec.ts.ejs

```diff
@@ -0,0 +1,55 @@
+<%#
+ Copyright 2019-Present the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see http://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { TranslateModule } from '@ngx-translate/core';
+import { NgxWebstorageModule } from 'ngx-webstorage';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { <%= entityAngularName %>Page } from './<%= entityFileName %>';
+
+
+describe('<%= entityAngularName %>Page', () => {
```

Comment:
Remove extra line break above this line.

Reply:
ok